### DISCUSSION TITLE
Aspect ratios

### DISCUSSION THREAD

**Discussion by Geo2D3D** [lllyasviel/Fooocus#155] [Ideas]
Hi! I searched within the *.json files and didn't find how to add aspect ratios. 
Is it, please, possible? If it's not could you think of adding that manual option?
Anyway, top job, thank you!

**Comment by gurasis** [comment]
Hi, I found the aspect ratios and styles in "\Fooocus\modules\sdxl_styles.py". 
You can prob add new aspect ratios in there at the bottom of the file.
![image](https://github.com/lllyasviel/Fooocus/assets/6304669/5bf2ad4b-8c51-4c9c-b0bf-01f387f90823)

**Comment by Geo2D3D** [comment]
Hi! Great! Thank you very much.
